### DISCUSSION TITLE
The partition method of train & test in load benchmark.

### DISCUSSION THREAD

**Discussion by robotcator** [polaris-hub/polaris#120] [General]
> Hi, @cwognum can you explain the split method when using get_train_test_split after load_benchmark?

refer [issue](https://github.com/polaris-hub/polaris/issues/115)

**Comment by cwognum** [comment]
Hi @robotcator , thanks for asking! 

Polaris does not impose a specific splitting method. Instead, the `BenchmarkSpecification` class simply [stores the indices](https://github.com/polaris-hub/polaris/blob/main/polaris/benchmark/_base.py#L102) associated with the train and test split. That way, users have full control over which splitting method they want to use. This is important, because which split is best depends on the downstream application you're trying to model (see e.g. [the MOOD paper](https://pubs.acs.org/doi/10.1021/acs.jcim.3c01774)). 

To help people propose benchmarks, we are working on and maintaining another package called [`splito`](https://github.com/datamol-io/splito) that implements several splitting methods relevant in drug discovery using a Scikit-learn interface. 

I hope that helps!
